src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -164,4 +189,19 @@ private function getTimezone(array $context)
 
         return $dateTimeZone instanceof \DateTimeZone ? $dateTimeZone : new \DateTimeZone($dateTimeZone);
     }
+
+    private function isForceContextTimezone(array $context): bool
+    {
+        $forceContextTimezone = $context[self::FORCE_CONTEXT_TIMEZONE] ?? $this->defaultContext[self::FORCE_CONTEXT_TIMEZONE];
```

Comment:
Can cast to `bool` here.

Reply:
Couldn't this whole method be rewritten as:

```suggestion
        if (null === $this->getTimezone($context)) {
            return false;
        }

        return (bool) $context[self::FORCE_CONTEXT_TIMEZONE] ?? $this->defaultContext[self::FORCE_CONTEXT_TIMEZONE];        
```

Reply:
Nice,  Done